### PR TITLE
Automate stress test eng-common-tools dockerfile release pipeline

### PR DESCRIPTION
**Pull request by ckairen**
Updating the docker release pipeline to automatically publish the dockerfile for stress test deployment container

### CODE REVIEW

File: eng/containers/ci.yml [hunk 1]
```diff
@@ -28,6 +28,13 @@ parameters:
     dockerFile: 'tools/stress-cluster/services/Stress.Watcher/Dockerfile'
     stableTags:
     - 'latest'
+  - name: stress_common_tools
+    pool: 'ubuntu-20.04'
+    dockerRepo: 'engsys/eng-common-tools'
```
Comment (benbp) [inline eng/containers/ci.yml:33]:
```suggestion
    dockerRepo: 'stress/common-tools'
```

Comment (benbp) [reply eng/containers/ci.yml:33]: @ckairen I would also move your deploy script into the stress-cluster directory alongside the Dockerfile and out of eng/common now that we're updating the approach.

Comment (ckairen) [reply eng/containers/ci.yml]: Dealt with here https://github.com/Azure/azure-sdk-tools/pull/2053

**Review by benbp — APPROVED** [review]

**Pull request merged**